src/main/java/com/iyzipay/request/CreateCardManagementPageInitializeRequest.java

```diff
@@ -67,6 +76,7 @@ public String toString() {
                 .appendSuper(super.toString())
                 .append("addNewCardEnabled", addNewCardEnabled)
                 .append("validateNewCard", validateNewCard)
+                .append("debitCardAllowed", debitCardAllowed)
```

Comment:
this field should be added to the end of append.

Reply:
done